bitnami/ghost/templates/ingress.yaml

```diff
@@ -25,9 +25,8 @@ spec:
       http:
         paths:
           - path: {{ default "/" .path }}
-            backend:
-              serviceName: "{{ include "common.names.fullname" $ }}"
-              servicePort: http
+            pathType: Prefix
```

Comment:
I'm reticent to add this new attribute and hardcode it. Would it work to have this implemented using a `.Values.ingress.pathType` parameter? You could use our implementation at [bitnami/apache](https://github.com/bitnami/charts/blob/6e457f514abf626f8eb57d4fdf5f681384623f24/bitnami/apache/templates/ingress.yaml#L31) for reference. If so, this could constitute a minor version instead of a patch, WDYT?